internal/benchmarks/package.json

```diff
@@ -0,0 +1,19 @@
+{
+  "private": true,
+  "name": "@internal/benchmarks",
+  "files": [
+    "dist"
+  ],
+  "scripts": {
+    "type:check": "tsc --noEmit",
+    "pretest": "run-s build:forc type:check",
+    "build:forc": "pnpm fuels build"
+  },
+  "license": "Apache-2.0",
+  "dependencies": {
+    "@fuel-ts/crypto": "workspace:*",
+    "@fuel-ts/account": "workspace:*",
```

Comment:
Redundant imports? Maybe we should also just benchmark the umbrella package?

Reply:
Good catch a0597b9d7